sources/Application/Model/Project.cpp

```diff
@@ -30,7 +30,10 @@ tempoNudge_(0)
 	this->Insert(transpose) ;
     Variable *softclip =
         new Variable("softclip", VAR_SOFTCLIP, softclipStates, 5, 0);
-    this->Insert(softclip) ;
+    this->Insert(softclip);
+    Variable *clipAttenuation =
+        new Variable("clipAttenuation", VAR_CLIP_ATTENUATION, 10);
```

Comment:
I think it would be better to have the default of "no attenuation", 100, and make it work regardless if softclip is turned on or not; I'd also put it after the hardclip part, since it should never cause any additional clipping, and I do drive the hardclip for effect, it works in small doses

Since it is now a de-facto master volume, and current master volume is drive/gain/etc. maybe it'd be better to rename current "master" to "drive"/"gain"/etc. and "post" to "master"/"out".

What do you think?

Reply:
I thought about it and figured this was the least intrusive way of doing it without risking affecting legacy projects. Didn't verify that's actually a problem though.

Reply:
Shouldn't affect legacy projects, we can bypass the logic entirely if it's at 100, so it should be exactly the same as 1.4.2 by default, so if 1.4.2 was fine for earlier projects then this should be too, I think :) I'll try run some older stuff over the weekend

Reply:
> Thanks a lot for your input. I have a lot of stuff going on rn and I'd like to free up some mental space. I'll keep this as is for now, it'll go out with the next beta build so people can help me catch any inevitable bugs that'll show up. Might refactor at some point.

Alright, sounds good! Same here about stuff going on, I might return to it myself and make a PR of some sort later